docs/client-sdk/javascript/tutorials/build-key-xmtp-chat-features-in-a-lens-app.md

```diff
@@ -81,7 +81,15 @@ const buildConversationId = (profileIdA: string, profileIdB: string) => {
 
 Now that you've built the Lens DM `conversationId` for your Lens app, your app can use it to filter and organize Lens conversations.
 
-You will filter for Lens conversations and then display their message previews.
+For example, you can use the Lens DM conversation ID as a filter to provide a UI in your app that displays Lens conversations only.
+
+:::tip Provide an "interoperable inbox"
+
+You might also want to provide a UI with no conversation ID filtering to display all conversations for a user. This enables your user to access and engage with all of their conversations, Lens and otherwise, without leaving your app. If you decide to provide a UI like this, you might want to use [conversation labeling](label-conversations).
+
```

Comment:
Not sure if I got the positioning right ... any thoughts? <3

Reply:
❤️